blue_dynamics/src/hydrodynamics.cpp

```diff
@@ -60,7 +60,7 @@ Coriolis::Coriolis(
   const Eigen::Vector6d & added_mass_coeff)
```

Comment:
The constructors should probably be updated to use `std::move` instead of passing a const ref.